### ISSUE TITLE
Tiny jupyter lab notebook toolbars

### ISSUE THREAD

**Issue by cwbeitel** [kubeflow/kubeflow#1704]
JupyterLab notebook toolbars appear hidden, CSS height set to 8px, toolbar returns when increasing this back to 30px or so.

<img width="704" alt="tinytoolbar" src="https://user-images.githubusercontent.com/2348401/46417583-db5acb00-c6de-11e8-9c20-acc89bbefd3e.png">

I believe this is is the 1.8 CPU build.

Looks like others have seen this and were able to fix by re-building
https://github.com/jupyterlab/jupyterlab/issues/593

cc @pdmack

**Comment by pdmack** [comment]
@richardsliu I imagine we will be doing more fresh builds of our images through the 0.3.0 release?

**Comment by jlewi** [comment]
Aren't the Jupyter images being rebuilt continuously now?

**Comment by pdmack** [comment]
I'm assuming we do `--no-cache` in the Docker builds?

**Comment by jlewi** [comment]
@pdmack I don't think we do. But we build using docker in docker running in Argo pods so I don't think there's any cache.

**Comment by cwbeitel** [comment]
I don't see this issue with at least gcr.io/kubeflow-images-public/tensorflow-1.10.1-notebook-cpu:v0.3.0

**Comment by jlewi** [comment]
@cwbeitel So if you aren't seeing this with v0.3.0 then what images were you seeing this with?

**Comment by cwbeitel** [comment]
I believe I was seeing this with gcr.io/kubeflow-images-public/tensorflow-1.6.0-notebook-cpu:v20180419-0ad94c4e but perhaps the default images changed upon re-deploying and that's not the one (launched it just now and don't see the issue there either). Given that I vote to close this issue.
